iteration.qmd

```diff
@@ -246,18 +250,19 @@ df |>
   )
 ```
 
-If you'd like to instead create new columns, you can use the `.names` argument give the output new names:
+If you'd like to instead create new columns, you can use the `.names` argument to give the output new names:
 
 ```{r}
 df |> 
   mutate(
-    across(a:d, \(x) x * 2, .names = "{.col}_double")
```

Comment:
I assume they've seen "double" in the context of double precision floating point numbers, so seems best to not risk confusion. Just do something else.